maquetado/scss/application.scss

```
@@ -1,2 +1,3 @@
 @import 'equalizer';
 @import 'styles';
+@import 'titles';
```

```suggestion
@import 'title';
```
Change the folder name for singular